store.go

```diff
@@ -407,6 +407,41 @@ func (c *StoreClient) ParseNotificationV2WithClaim(tokenStr string) (jwt.Claims,
 	return result, err
 }
 
+func (c *StoreClient) ParseSignedPayload(tokenStr string, claims jwt.Claims) error {
```

Comment:
It seems this method is used internal, maybe lowercase is better. `parseSignedPayload`

Reply:
I left this public on purpose, since otherwise there is no way to parse any of the signed payloads into custom claims objects.

Reply:
Got it. Thank you for your explanation.   :thumbsup: